scripts/install_third_party.sh

```diff
@@ -19,7 +19,7 @@ source $(dirname $0)/setup.sh || exit 1
 
 # Download and install required JS and zip files.
 echo Installing third-party JS libraries and zip files.
-python scripts/install_third_party.py
+$PYTHON_CMD scripts/install_third_party.py
```

Comment:
I think you should add the debug echo's here, as well. The latest Travis CI build failed before running install_third_party.py this time.